src/components/tx/modals/BatchExecuteModal/ReviewBatchExecute.tsx

```diff
@@ -65,61 +65,59 @@ const ReviewBatchExecute = ({ data, onSubmit }: { data: BatchExecuteData; onSubm
   const submitDisabled = loading || !isSubmittable
 
   return (
-    <div>
```

Comment:
This wrapper is needed with TxModal for the spacing. We could refactor this separately though.
<img width="632" alt="Screenshot 2022-09-02 at 09 43 15" src="https://user-images.githubusercontent.com/5880855/188085167-0f765581-a5ba-4045-b5e7-eacb72bdc5cc.png">